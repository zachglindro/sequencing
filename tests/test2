2 5
2 10 3 1
3 4 4 0