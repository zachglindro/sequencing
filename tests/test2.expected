Solution found. Number of S tasks completed: 1
Optimal Schedule: 0 -> 1